### DISCUSSION TITLE
Cloned repo has lots of file that I can't see at github.com

### DISCUSSION THREAD

**Discussion by bdnstn** [community/community#126319] [New to GitHub]
When I clone (or download source) from https://github.com/merbanan/rtl_433_tests I get a lot of folders (and files) that are not visible in github. 
Interestingly, if I search _rtl_433_tests_ in github it says it was **updated 22 days ago**, but when I go to the repo it says the **last commit was 5 months ago**. I have "master" selected as the branch.
I've cleared browser cache, signed out, tried a different browser. 
Can you suggest what's going on here?
Thanks, in anticipation!
Brian

**Comment by unistar18** [comment]
Hi,
Check the .gitignore file.
```
*~
__pycache__
```
This repo is exactly last changed in 5 months ago.


or this can sometimes happen when GitHub's content delivery network (CDN) serves cached versions of the repository that may not be up to date. This can result in discrepancies between what you see when you clone or download the repository and what is displayed on the GitHub website.
Try to clear your browser cache, signing again.

**Reply by bdnstn** [reply]
Thanks for the suggestion. I've already tried clearing browser cache.
It isn't a .gitignore issue. I'm not missing particular types of files, it's like there are 20% of the test folders are missing.

**Comment by pedrolemos4** [comment]
Maybe those files are not being tracked by git anymore and where changed manually in the master branch through GitHub
